src/__tests__/AvatarStack.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React from 'react'
+import AvatarStack from '../AvatarStack'
+import {render} from '../utils/testing'
+import {COMMON} from '../constants'
+
+const avatarComp = (
+  <AvatarStack>
+    <AvatarStack.Item src="https://avatars.githubusercontent.com/primer" />
```

Comment:
should the tests just be updated to use `<img />` tags now?

Reply:
Oh yeah my bad!